tests/test-compile-large-binary.cc

```diff
@@ -0,0 +1,18 @@
+/*
+ * This should compile to a 64MB binary with the array variable
+ * encoded in the binary.
+ *
+ * @EXPECTED_RESULTS@: WRONG-ANSWER
+ */
+
+#include <stdio.h>
+
+const int N = 64 * 1024 * 1024;
+
+char array[N] = {1};
+
+int main()
+{
+	printf("Successfully compiled with a %d MB static array.\n", N / 1024 / 1024);
```

Comment:
Because array is never used, should it not be sizeof(array) because otherwise this can be removed by the compiler?

Reply:
it's statically initialized and thus can't be removed

Reply:
For clarity I've used `sizeof` while making some other small fixes.